meson.build

```diff
@@ -83,14 +83,8 @@ else
   pkg_prefix = option_pkg_prefix
 endif
 
-if get_option('split-usr') == 'auto'
-  split_usr = run_command('test', '-L', '/bin', check: false).returncode() != 0
-else
-  split_usr = get_option('split-usr') == 'true'
-endif
-
 rootprefix = get_option('rootprefix')
-rootprefix_default = fs.is_symlink('/bin') ? '/usr' : '/'
+rootprefix_default = '/'
 if rootprefix == ''
 rootprefix = rootprefix_default
 endif
```

Comment:
Mysteriously, this autodetection is always overridden by gentoo by passing `-Drootprefix="${EPREFIX}"`, which leaves me wondering where the actual source of the linked bug is.

Reply:
> Mysteriously, this autodetection is always overridden by gentoo by passing -Drootprefix="${EPREFIX}", which leaves me wondering where the actual source of the linked bug is.

That only affects line 86. On a normal system, ${EPREFIX} is empty so rootprefix is also empty.

The condition on line 94 is satisfied, so line 95 sets rootprefix = rootprefix_default.

rootprefix_default is the auto-detected value based on /bin.

Reply:
It could be done, but I don't plan on doing it myself. I've updated the commit message a bit.